.changeset/gorgeous-poets-cry.md

```diff
@@ -0,0 +1,6 @@
+---
+"@commercetools-uikit/data-table-manager": minor
+"@commercetools-uikit/data-table": minor
```

Comment:
this was added automatically, not sure if I need to leave it or remove, cause I didn't seem to change anything in the DataTable itself.